### DISCUSSION TITLE
Bienvenido al lugar de debate sobre Provida ngDAO

### DISCUSSION THREAD

**Discussion by IBaixauli** [NGDAOorg/provida#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by IBaixauli** [comment]
¿Qué te parece este sistema de discusiones?

**Comment by ruth-ga** [comment]
Muy necesario e interesante sobre un tema que NO  tendría que ser polémico por los derechos inherentes que tenemos como seres humanos.  Ser pro-vida  o no serlo es lo que nos define como seres humanos. (valga la redundancia)

**Reply by ruth-ga** [reply]
Muy necesario e interesante sobre un tema que NO tiene que ser polémico por los derechos inherentes que tenemos como seres humanos. Ser pro-vida o no serlo es lo que nos define como seres humanos. (valga la redundancia)
